go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -246,13 +232,12 @@ func (npw *nodePortWatcher) updateServiceFlowCache(service *kapi.Service, add, h
 // case1: If a service has externalTrafficPolicy=local, and has host-networked endpoints local to the node (hasLocalHostNetworkEp),
 // traffic instead will be steered directly into the host and DNAT-ed to the targetPort on the host.
 //
-// case2: Only applicable for LGW mode: If a service has externalTrafficPolicy=local, and it doesn't have host-networked
-// endpoints local to the node (!hasLocalHostNetworkEp - including empty endpoint.Subset), traffic will be steered into LOCAL
-// preserving sourceIP and IPTables will steer this traffic into OVN via ovn-k8s-mp0.
-//
-// case3: All other types of services i.e:
+// case2: All other types of services i.e:
 //        case3a: if externalTrafficPolicy=cluster, irrespective of gateway modes, traffic will be steered into OVN via GR.
```

Comment:
this isn't true anymore right?

Reply:
yeah good catch again, let me update the comment.